### DISCUSSION TITLE
connectkit switch network doesnt work on wagmi 0.7.x

### DISCUSSION THREAD

**Discussion by johnpaulcas** [family/connectkit#46] [Q&A] [answered]
I upgraded my library from 0.6 to 0.7 and now the conneckit switch network doesnt work on wagmi 0.7, its all working before on 0.6, may I ask what configuration I need to set on connectkit? thanks 🙏

**Comment by lochie** [comment, marked as answer]
ConnectKit version `1.1.0` now works with the latest version of wagmi `0.8.x`, we've skipped wagmi version `0.7.x` since it was a short lived minor version.

**Reply by johnpaulcas** [reply]
thanks @lochie, really appreciate the efforts 🙏 🚀
